src/components/Codebox/index.tsx

```diff
@@ -34,24 +35,9 @@ const Codebox = ({ children: { props } }: Props): JSX.Element => {
 
   const handleCopyCode = (event: React.MouseEvent<HTMLButtonElement>) => {
     event.preventDefault();
-    navigator.clipboard.writeText(stringCode);
-    setCopied(true);
+    copy(stringCode);
```

Comment:
Maybe the return value of `copy` could be a boolean that tells if the content got copied. Instead of a variable called `copied`